tools/rpkg/NAMESPACE

```diff
@@ -11,10 +15,20 @@ export(duckdb_register_arrow)
 export(duckdb_shutdown)
 export(duckdb_unregister)
 export(duckdb_unregister_arrow)
+export(expr_constant)
+export(expr_function)
+export(expr_reference)
+export(expr_tostring)
 export(read_csv_duckdb)
+export(rel_aggregate)
+export(rel_explain)
+export(rel_filter)
+export(rel_from_df)
+export(rel_order)
+export(rel_project)
+export(rel_sql)
 exportClasses(duckdb_connection)
 exportClasses(duckdb_driver)
-exportClasses(duckdb_explain)
```

Comment:
No need to remove this, please re-run `roxygen` before merge.

Reply:
Thanks for spotting, no longer removed. Not sure what happened there.